src/components/Filter/FilterHelper.spec.tsx

```diff
@@ -0,0 +1,468 @@
+import { createFilters, sanitizeParams, paramsFromFilter } from './FilterHelper';
+import { filterOptions } from './__mocks__';
+import { FilterType, AdvancedFilterCategoryModel } from '@osu-cass/sb-components';
+
+describe('FilterHelper.createFilters', () => {
+  it('grade no params', () => {
+    const params = { grades: [] };
+    const result = createFilters(filterOptions, params);
+
+    const expectation: AdvancedFilterCategoryModel = {
```

Comment:
Since we have a lot of the same object with slightly different parameters, the jest convention to do this is to use the `beforeEach` to initialize each test object each time to some default values, then inside the test we would just change whatever properties that will change the outcome of the test. This would decrease the lines of code in this file and make it more clear what exact part of the filter you're testing.